src/model/table.py

```diff
@@ -19,12 +21,32 @@ def get_name(self):
     def get_path(self):
         return self.__path__
 
-    # Will be implemented later in the project
-    def set(self):
-        pass
-
-    def delete(self):
-        pass
+    def get_data_path(self):
+        return os.path.join(self.__path__, "data")
+        
+    def get_primary_key(self):
+        return self.__table_metadata__.primary_key
+
+    def can_overwrite(self):
+        return eval(self.__table_metadata__.overwrite)
+
+    def get_indices(self):
+        return self.__table_metadata__.index_keys
+
+    def set(self, data):
```

Comment:
Set it table should take Row instance.

Who create the row is database. To let the table deals with row only